pkg/services/accesscontrol/models.go

```diff
@@ -269,12 +273,13 @@ type SetResourcePermissionCommand struct {
 }
 
 const (
-	GlobalOrgID        = 0
-	FixedRolePrefix    = "fixed:"
-	ManagedRolePrefix  = "managed:"
-	BasicRolePrefix    = "basic:"
-	BasicRoleUIDPrefix = "basic_"
-	RoleGrafanaAdmin   = "Grafana Admin"
+	GlobalOrgID         = 0
+	FixedRolePrefix     = "fixed:"
+	ManagedRolePrefix   = "managed:"
+	BasicRolePrefix     = "basic:"
+	AppPluginRolePrefix = "plugins.app:"
```

Comment:
question: this is potential bike-shedding, but are we set on the `plugins.app` prefix? Why not just `plugins`?

Reply:
Yeah, I wonder the same :smile:

Reply:
Not set on it, but consistent with the permissions we used for app plugins access.
@marefr I remember you suggested `apps` as well?